docroot/user/themes/lucysanchez/templates/modular/hero.html.twig

```diff
@@ -0,0 +1,18 @@
+<section class="hero-intro">
```

Comment:
In future template it would be good to use conditionals to validate the variable has a value. (This case is not to necessary)